css/main.css

```diff
@@ -468,30 +459,27 @@ footer {
 @media screen and (min-width: 500px) {
   .description-what-is-title {
     font-size: 2.8rem;
-    padding-top: 6%;
-    padding-bottom: 3%;
-    padding-left: 15%;
-    padding-right: 15%;
+    padding: 90px 60px 45px;
   }
 
   .description-what-is-content {
     font-size: 1.25rem;
-    padding: 0 20%;
-    padding-bottom: 6%;
+    padding: 0px 300px 90px;
   }
 
   .description-why-join-title {
     font-size: 2.8rem;
-    margin-top: 10%;
+    margin-top: 150px;
+    margin-bottom: 100px;
```

Comment:
This can also be written in one line: `margin: 150px 0px 100px` This would make the left and right margin 0px automatically and set the top and bottom margin to 150px and 100px. respectively.

Reply:
Got it.